jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+};
```

Comment:
We should add a newline here!

Reply:
`npx ts-jest config:init` disagrees, but I will add it :)

Reply:
👑